docs/modules/ROOT/pages/includes/getting-started.adoc

```diff
@@ -95,6 +95,20 @@ You can remove operationId prefix (e.g. User_findAll=> findAll). To do that, you
 quarkus.openapi-generator.codegen.spec.petstore_json.remove-operation-id-prefix=true
 ----
 
+Character to use as a delimiter for the prefix. Default is '_'.You can define the prefix delimiter (e.g. User.findAll=> findAll):
```

Comment:
Can we have a test case to validate this?

Reply:
yes.

https://github.com/quarkiverse/quarkus-openapi-generator/blob/3f091923eac373c9d16b918e3c9d972444114cd8/client/integration-tests/remove-operationid-prefix/src/main/resources/application.properties#L9

https://github.com/quarkiverse/quarkus-openapi-generator/blob/3f091923eac373c9d16b918e3c9d972444114cd8/client/integration-tests/remove-operationid-prefix/src/main/openapi/openapi-prefix-delimiter.yaml#L17

https://github.com/quarkiverse/quarkus-openapi-generator/blob/3f091923eac373c9d16b918e3c9d972444114cd8/client/integration-tests/remove-operationid-prefix/src/test/java/io/quarkiverse/openapi/generator/it/RemoveOperationIdPrefixTest.java#L45